manifest.yaml

```diff
@@ -69,24 +46,10 @@ volumes:
     volume-id: main
     path: "/public"
     readonly: true
-interfaces:
-  main:
-    name: Network Interface
-    description: Specifies the interface to listen on for HTTP connections.
-    tor-config:
-      port-mapping:
-        80: "80"
-    lan-config:
-      443:
-        ssl: true
-        internal: 80
-    ui: false
-    protocols:
-      - tcp
-      - http
+interfaces: {}
```

Comment:
Why did you delete the interface?

Reply:
Jesse opened an issue for it. Jet doesn't use the interface at all, its strictly a local CLI tool

Reply:
What was the problem, and how does this fix?

Reply:
https://github.com/Start9Labs/lightning-jet-wrapper/issues/1

Reply:
Yes, I already looked at it. The reason I asked the question is because it doesn't say what the problem was or what the fix is.